charts/aws-efs-csi-driver/templates/node-daemonset.yaml

```diff
@@ -50,28 +50,25 @@ spec:
                 values:
                 - fargate
       hostNetwork: true
-      {{- if .Values.dnsPolicy }}
-      dnsPolicy: "{{ .Values.dnsPolicy }}"
-      {{- end }}
-      {{- with .Values.dnsConfig }}
+      dnsPolicy: {{ .Values.node.dnsPolicy }}
+      {{- with .Values.node.dnsConfig }}
       dnsConfig:
         {{- toYaml . | nindent 8 }}
       {{- end }}
       priorityClassName: system-node-critical
-      tolerations:
-        - operator: Exists
```

Comment:
The daemonset and the deployment should behave differently.  Removing the operator: Exists from the daemonset I think would be surprising to users.  I think a daemonset needs some sort of default tolerations since it is supposed to be running on "every" node.  I think a reasonable solution might be to have the operator: Exists as a default in the values.yaml and then if a user wants they have a method of getting rid of that toleration.

```
node:
  ...
  tolerations:
    - operator: Exists
```

Reply:
Fixed: https://github.com/kubernetes-sigs/aws-efs-csi-driver/pull/406/commits/a2aa0794e2ddd4921e068fc044d201a67e86b662